Azure/sonic-frr
# Support SONiC debian8 on FRR 4.0.x

- What I did
Make FRR 4.0 buildable with SONiC debian8 dev environment.

- How I did it
Add sonic.debian8 target with proper settings for SONiC

- How to verify it
Build with sonic-buildimage (PR TBD), and load the image. FRR starts fine, and also FPM worked as expected towards SONiC :

admin@sonic:~$ vtysh

Hello, this is FRRouting (version 4.0).
Copyright 1996-2005 Kunihiro Ishiguro, et al.

sonic# show running-config 
Building configuration...

Current configuration:
!
frr version 4.0
frr defaults traditional
hostname sonic
service integrated-vtysh-config
!
log syslog informational
!
log facility local4
!
router bgp 65100
 bgp router-id 10.1.0.1
 bgp log-neighbor-changes
 no bgp default ipv4-unicast
 
....

sonic# configure terminal
sonic(config)# ip route 10.10.10.0/24 10.0.0.17

redis-cli:

127.0.0.1:6379> hgetall ROUTE_TABLE:10.10.10.0/24

"nexthop"
"10.0.0.17"
"ifname"
"Ethernet8"

More tests will be done at sonic-buildimage PR.

- Description for the changelog
